builtin/logical/consul/path_roles.go

```diff
@@ -71,6 +71,20 @@ Defaults to 'client'.`,
 				Description: "Use ttl instead.",
 				Deprecated:  true,
 			},
+
+			"consul_namespace": {
```

Comment:
Do we need "consul_" here?

Reply:
My thought was to be verbose to avoid potential confusion when using namespaces in the CLI command. I considered that
`vault read -namespace="ns1" consul/creds/my-role namespace="ns2"`
could be mistakenly read and cause someone to mix up which namespace was vault's and which namespace was consul's. So instead, it would be
`vault read -namespace="ns1" consul/creds/my-role consul_namespace="ns2"`

I'm interested in any other opinions or ideas that might be a better solution.

Reply:
sgtm